frontend/test/metabase-smoketest/admin.cy.spec.js

```diff
@@ -160,6 +160,7 @@ describe("metabase-smoketest > admin", () => {
     });
 
     it("should add a simple JOINed question as admin", () => {
+      cy.visit("/question/new");
       cy.findByText("Ask a question");
 
       cy.findByText("Ask a question").click();
```

Comment:
I just noticed this. It seems redundant. But anyway, it's not a focus of this PR - just a mental note to me.